CHANGELOG.asciidoc

```diff
@@ -7,6 +7,146 @@
 === Beats version 8.16.6
 https://github.com/elastic/beats/compare/v8.16.5\...v8.16.6[View commits]
 
+==== Known issues
+
+- `restart_on_cert_change` causes panic due to seccomp policy. In versions 8.16.6 and later, enabling this option causes the Beat to panic on restart. This is due to the `eventfd2` syscall missing from the default seccomp policy. To fix this, add `eventfd2` to a custom seccomp policy. For more details, refer to <<linux-seccomp>>.
```

Comment:
Oh no... I've already forgotten how to write AsciiDoc! 🙈  I can update the other PRs and babysit them to make sure the build succeeds. 

```suggestion
- `restart_on_cert_change` causes panic due to seccomp policy. In versions 8.16.6 and later, enabling this option causes the Beat to panic on restart. This is due to the `eventfd2` syscall missing from the default seccomp policy. To fix this, add `eventfd2` to a custom seccomp policy. For more details, refer to {filebeat-ref}/linux-seccomp.html[Use Linux Secure Computing Mode (seccomp)].
```

Reply:
ok then. I added the known issue for every patch on every minor PR. but the link to `linux-seccomp` is still broken.

if you need help, let me know